pkg/projectfile/projectfile.go

```diff
@@ -664,7 +664,7 @@ func (p *Project) parseURL() (projectURL, error) {
 
 func validateUUID(uuidStr string) error {
 	if ok := strfmt.Default.Validates("uuid", uuidStr); !ok {
-		return locale.NewError("invalid_uuid_val", "Invalid commit ID {{.V0}} in activestate.yaml. Please remove it and run `[ACTIONABLE]state pull[/RESET]` to reset it", uuidStr)
+		return locale.NewError("err_commit_id_invalid", "", uuidStr)
```

Comment:
This key is already defined in en-us.yaml, so there's no need for the optional text.